lib/widgets/cards/notification_card.dart

```diff
@@ -60,15 +58,15 @@ extension NotificationTypeExtension on NotificationType {
       GestureDetector(
         onTap: () => {log("open the grading page")},
         child: AttributeChip.withIcon(
-          content: AppLocales.of(context).translate("$_pageKey.caregiver.gradeTask"),
```

Comment:
Why did you remove the _pageKey? I know it's not a page but you can call it _cardKey for example and it's better with than without it

Reply:
Because I used it in both the enum extension and the card class. I thought that if I left the constant in one place that if someone wanted to change it, he will overlook the other.

Reply:
It's still used, just passed in different place. To me all those texts belong together (as they are about the same thing) so if they will be moved, they will get moved together anyway.. That constant makes moving it easier